File: CS/MyDataSourceWrapper/MyObject.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using System.Collections;

namespace WindowsApplication1
{
    public class MyObject
    {

        public MyObject()
        {

        }

        private int _ID;
        public int ID
        {
            get { return _ID; }
            set
            {
                _ID = value;
            }
        }

        private object _DisplayText;
        public object DisplayText
        {
            get { return _DisplayText; }
            set { _DisplayText = value; }
        }
    }
}

File: VB/MyDataSourceWrapper/MyObject.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports System.Collections

Namespace WindowsApplication1
	Public Class MyObject

		Public Sub New()

		End Sub

		Private _ID As Integer
		Public Property ID() As Integer
			Get
				Return _ID
			End Get
			Set(ByVal value As Integer)
				_ID = value
			End Set
		End Property

		Private _DisplayText As Object
		Public Property DisplayText() As Object
			Get
				Return _DisplayText
			End Get
			Set(ByVal value As Object)
				_DisplayText = value
			End Set
		End Property
	End Class
End Namespace
